### PR TITLE
Temporary fix for loading miniconda in a bash script if the login she…

### PR DESCRIPTION
**Pull request by JeffBeck-NOAA**
…ll is tcsh

### CODE REVIEW

File: ush/load_modules_run_task.sh [hunk 1]
```diff
@@ -298,6 +298,7 @@ Call to \"module use\" command failed."
      # files, we source the "module" file.  For true module files, we use the
      # "module load" command.
      #
+     SHELL="/bin/sh"
```
Comment (christinaholtNOAA) [inline ush/load_modules_run_task.sh]: I think this should be `SHELL="/bin/bash"`.

**Comment by JeffBeck-NOAA** [comment]
On Jet, Hera, and Cheyenne, /bin/sh points to bash, but we could use
/bin/bash if that makes more sense.

**Comment by christinaholtNOAA** [comment]
The first line of this script indicates that this is a `/bin/bash` shell explicitly.

**Comment by JeffBeck-NOAA** [comment]
OK, I was just using what was in Gerard's email.  I can change it and push
a commit.

- Jeff

**Comment by christinaholtNOAA** [comment]
I believe I have fixed this issue in the module. I have asked Gerard to test it out. If it works, this PR will no longer be necessary.

**Comment by JeffBeck-NOAA** [comment]
Thanks, Christina.  Sounds good.

**Comment by gsketefian** [comment]
Christina:

Your new "module load conda" works on hera (after launching a bash shell from my tcsh login shell).  Questions:
1) What did you do to fix it?
2) Did you make the same fix on any other machines?

Thanks!
Gerard

**Comment by christinaholtNOAA** [comment]
@gsketefian That's great news! I used the TCL command that is more reliable than using the SHELL env variable:

```
- set shell $env(SHELL)
+ set shell [module-info shelltype]
```

I was waiting on your confirmation, but will now make the Jet modulefile identical.

**Comment by gsketefian** [comment]
@christinaholtNOAA Ok, sounds good, thank you!

**Comment by christinaholtNOAA** [comment]
Can we close this one?

**Comment by gsketefian** [comment]
@JeffBeck-NOAA I'm closing out this PR.  Please reopen if you think there's something else to address.

**Comment by JeffBeck-NOAA** [comment]
Sounds good.  Thanks!

**Pull request closed without merging**